src/autoload.php

```diff
@@ -19,3 +27,16 @@ class_alias(ContainerExceptionInterface::class, InteropContainerException::class
 if (! interface_exists(InteropNotFoundException::class, false)) {
     class_alias(NotFoundExceptionInterface::class, InteropNotFoundException::class);
 }
+
+$installedContainerVersion = InstalledVersions::getVersion('psr/container');
+
+assert(
+    $installedContainerVersion !== null,
+    'psr/container is required by `composer.json` and therefore this method should not return `null`.'
```

Comment:
"this method" is a bit out of context now :D 

Anyway, not a problem, just noticed it as a tiny detail